core-httpclient-impl/src/main/java/com/optimizely/ab/OptimizelyFactory.java

```diff
@@ -189,6 +189,25 @@ public static Optimizely newDefaultInstance(String sdkKey, String fallback) {
         return newDefaultInstance(builder.build(), notificationCenter);
     }
 
+    /**
+     * Returns a new Optimizely instance with authenticated datafile support.
+     *
+     * @param sdkKey   SDK key used to build the ProjectConfigManager.
+     * @param authDatafileToken  Token for authenticated datafile access.
+     * @param fallback Fallback datafile string used by the ProjectConfigManager to be immediately available.
+     */
+    public static Optimizely newDefaultInstance(String sdkKey, String authDatafileToken, String fallback) {
```

Comment:
In other SDKs we named it `datafileAuthToken`, could you please follow that for all variable & method names?